docs/source/models/intervention_models/intrapartum/acs_intervention.rst

```diff
@@ -0,0 +1,235 @@
+
+This intervention requires adding an attribute to all simulants who expect to give birth to a preterm infant (i.e., based on believed gestational age 34 to 36 weeks from pregnancy module output) and who give birth 
+in a facility with CPAP availability (based on ``cpap_availability``; see :ref:`the CPAP intervention document <intervention_neonatal_cpap>`) to specify if a parent-child dyad
+receives ACS or not.  We will track this and the model will have different mortality rates for preterm with RDS for parent-child dyads with and without 
+ACS (implemented with a slightly confusing application of our ``Risk`` and ``RiskEffect`` components from ``vivarium_public_health``).
+
+The ``Risk`` component adds an attribute to each simulant indicating whether the simulant has received ACS during the intrapartum period. Only simulants who expect to 
+give birth to a preterm infant and who give birth in a facility with CPAP availability are eligible for this intervention.
+
+To make this work naturally with the ``RiskEffect`` component, it is best to think of the risk as "no ACS".  With this framing, the ``RiskEffect`` 
+component requires data on (1) the relative risk of preterm with RDS mortality for people who did not receive ACS, and (2) the population attributable fraction (PAF) of preterm with
+RDS deaths due to not receiving ACS.  We will use the decision tree below to estimate the probability of preterm with RDS mortality with and without the use of ACS, ensuring consistency
+with the baseline delivery facility rates and baseline ACS coverage.
+
+In Vivarium, this risk effect will modify the preterm with RDS mortality pipeline, resulting in 
+
+.. math::
+
+   \text{CSMRisk}_i^\text{preterm with RDS} = \text{CSMRisk}^\text{preterm with RDS} \cdot (1 - \text{PAF}_\text{no ACS}) \cdot \text{RR}_i^\text{no ACS}
+
+where :math:`\text{RR}_i^\text{no ACS}` is simulant *i*'s individual relative risk for "no ACS", meaning :math:`\text{RR}_i^\text{no ACS} = \text{RR}_\text{no ACS}` 
+if simulant *i* does not receive ACS, and :math:`\text{RR}_i^\text{no ACS} = 1` if simulant *i* receives ACS. 
+
+The relative risk value we will use is pulled from [Oladapo-et-al-2020]_, a BMGF-funded multicountry RCT which investigated the impact
+of ACS for pregnant women and people at imminent risk of preterm delivery.
+
+
+.. list-table:: Risk Effect Parameters for No ACS
+  :widths: 15 15 15 15
+  :header-rows: 1
+
+  * - Parameter
+    - Value
+    - Source
+    - Notes
+  * - :math:`\text{RR}^\text{no ACS}`
+    - :math:`1/\text{RR}^\text{ACS}`
+    - N/A
+    - Value to be used in sim
+  * - :math:`1/\text{RR}^\text{ACS}`
+    - RR = 0.84 (95% CI 0.72-0.97). Parameter uncertainty implemented as a lognormal distribution: :code:`get_lognorm_from_quantiles(0.84, 0.72, 0.97)`
+    - [Oladapo-et-al-2020]_
+    - 
+  * - mean_rr
+    - :math:`\text{RR}^\text{no ACS} * (1 - p_\text{baseline coverage}) + p_\text{baseline_coverage}`
+    - N/A
```

Comment:
I think we will need to use the more complicated PAF derivation that you've used in the neonatal interventions given that baseline coverage varies by delivery facility. We've been able to use this simplified equation when we have just one estimate of baseline coverage among the entire eligible population like for misoprostol and neonatal antibiotics.